packages/lit-virtualizer/src/LitVirtualizer.ts

```diff
@@ -113,11 +113,14 @@ export class LitVirtualizer extends LitElement {
     render(): TemplateResult {
         const { items, _renderItem, keyFunction } = this;
         const itemsToRender = [];
+        const lastItem = Math.min(items.length, this._last + 1);
+
         if (this._first >= 0 && this._last >= this._first) {
-            for (let i = this._first; i < this._last + 1; i++) {
+            for (let i = this._first; i < lastItem; i++) {
```

Comment:
I think this needs to be fixed here too: https://github.com/PolymerLabs/uni-virtualizer/blob/d90cbcc4430abf45abba007105ae2ec482eefbde/packages/lit-virtualizer/src/virtualize.ts#L66

Reply:
Yes—the directive and the element used to share a single implementation and we may be able to get back to that state eventually, but for now this logic needs to be maintained in both places.